lib/chef/config.rb

```diff
@@ -55,7 +56,7 @@ class Config
 
     default :event_loggers do
       evt_loggers = []
-      if ChefConfig.windows? && !Chef::Platform.windows_nano_server?
+      if ChefUtils.windows? && !Chef::Platform.windows_nano_server?
```

Comment:
We really need to remove all the nano checks in Chef 16. They killed off nano entirely.

Reply:
yep but that's not here